zkevm-circuits/src/super_circuit.rs

```diff
@@ -342,6 +398,7 @@ impl<F: Field> SubCircuit<F> for SuperCircuit<F> {
         let pi = PiCircuit::min_num_rows_block(block);
 
         let rows: Vec<(usize, usize)> = vec![evm, state, bytecode, copy, keccak, tx, exp, pi];
+        println!("rows {:?}", rows);
```

Comment:
yeah, I am still debugging the error occur during integration test, so there are some left over, I will clean up it sooner.

Reply:
Found the root cause of the integration test failed:

Because for fingerprint equality gate in super_circuit, the fingerprint columns are assigned in evm/state circuit region respective, so this gate can't be satisfied. 
I am currently thinking about how to resolve it. @davidnevadoc do you have any better idea~?

Reply:
> Found the root cause of the integration test failed:
> 
> Because for fingerprint equality gate in super_circuit, the fingerprint columns are assigned in evm/state circuit region respective, so this gate can't be satisfied. I am currently thinking about how to resolve it. @davidnevadoc do you have any better idea~?

This is solved by stealing expression from different region (this tricks we used in other place) 
Fixed in this commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1674/commits/c7c1f9794c17d7d0ae4cc1596ccd72edf77bfc1e